src/test/scala/io/iohk/ethereum/network/PeerManagerSpec.scala

```diff
@@ -286,6 +286,45 @@ class PeerManagerSpec
     }
   }
 
+  it should "try to connect to discovered nodes if there's an outgoing demand: new nodes first, retried last" in new TestSetup {
```

Comment:
Just for my understanding, what is the value of `connectedPeers.outgoingConnectionDemand` during this test?

Reply:
`maxOutgoingPeers` is set to 30, `outgoingConnectionDemand` is calculated based on that number minus number connections established. So we start with 30